Makefile

```diff
@@ -15,6 +15,22 @@ deploy-production: ## Deploy to prometheus-production
 	cf target -s prometheus-production
 	cf push -f manifest-production.yml
 
+.PHONY: update-service-broker-staging
+update-service-broker-staging:
+	# Note: you need to be a cloud foundry admin to run this task
+	scripts/update-service-broker.sh \
+		prometheus-staging \
+		prometheus-service-broker-staging \
+		prometheus-service-broker-staging
+
+.PHONY: update-service-broker-production
+update-service-broker-production:
+	# Note: you need to be a cloud foundry admin to run this task
+	scripts/update-service-broker.sh \
+		prometheus-production \
+		prometheus-service-broker \
+		prometheus-alpha
```

Comment:
Could you enlighten me to where the choice of service broker name `prometheus-alpha` (and `prometheus-service-broker-staging` for the `update-service-broker-staging` task) have come from please?

When I run `cf service-brokers` in any space I get

```
prometheus-service-broker-dev   
prometheus-service-broker-staging
prometheus-service-broker-test
```

Does this mean that the `prometheus-alpha` service broker is one that does not exist in our account and is instead managed by the PaaS team (and hence can appear in `cf marketplace`).

Reply:
Assuming there are no consequences of doing so and it's really quick then feel free too but I wouldn't spend more than 30 seconds on it as this code will probably be rarely looked at (or maybe that is a reason to do it). I'll let you decide, I'm happy either way.

Reply:
I don't really know what the consequences might be, so I think let's leave it for now.